databases/mongo-db.md

```diff
@@ -94,6 +94,8 @@ If no Homebrew-related text appears, you'll need to install Homebrew. Go to the
   - [Start here](https://www.tutorialspoint.com/mongodb/mongodb_data_modeling.htm) and read until the Deployment section.
   - [Start here](https://www.tutorialspoint.com/mongodb/mongodb_relationships.htm) and read until the Regex section.
 2. Follow [this video tutorial](https://youtu.be/4yqu8YF29cU) for building a full Mongo project on your machine with an mLab online mongo database. This will take at least 2 hours, so settle in! If you don't feel confident that you understand the parts of the project he is going over, stop the video and look up more examples.
+ **Instructions**
```

Comment:
1. Please check the preview and ensure this starts on a new line.
2. Please change “instructions” to “note”
3. Please make this shorter! Tell them concisely what do do. Then tell them why.